core/app/models/spree/address.rb

```diff
@@ -148,6 +148,10 @@ def readonly?
       persisted?
     end
 
+    def country_iso=(iso)
```

Comment:
Can we get a quick yardoc on this, just to clarify what it does (not a standard setter).

Otherwise :+1: